Inception/Inception/Network/Models/SleepRecord.swift

```diff
@@ -35,4 +35,10 @@ struct SleepRecord: Codable {
   var trackedDate: String {
     return Date().dateToStringMMDD(wakeuptimeDate)
   }
+  
+  init(sleepSatisfacation: SleepSatisfacation, bedtimeDate: Date, wakeuptimeDate: Date){
```

Comment:
```suggestion
  init(sleepSatisfacation: SleepSatisfacation, bedtimeDate: Date, wakeuptimeDate: Date) {
```

p3;

소괄호와 중괄호 사이 띄어쓰기가 하나 빠진 거 같아요~

Reply:
8a4f3a1ab8aaf8ae477579139be3888f7b0fc8ed
오 매의눈...! 수정했습니다!